### ISSUE TITLE
theme: palettes using outdated opacity for light contrast

### ISSUE THREAD

**Issue by clshortfuse** [angular/material#8992]
**Actual Behavior**:
-  `What is the issue?` Opacities for light contrast vary between 87% and 100%
-  `What is the expected behavior?`Opacity for light contrast should be fixed to 100%

**Angular Versions**: *
-  `Angular Version:` N/A
-  `Angular Material Version:` master

Previously, the spec stated opacities were set to 87% for text on dark background
https://web.archive.org/web/20140628054353/http://www.google.com/design/spec/style/color.html#color-ui-color-application

Now, it's 100% always:
https://material.google.com/style/color.html#color-text-background-colors

**Comment by Splaktar** [comment]
The spec updates for 2017 can still be found here:
https://material.io/archive/guidelines/style/color.html#color-usability

This archive holds the details prior to the MD 2018 changes.

**Comment by Splaktar** [comment]
I'm going to close this as I think that it is fully covered by #10164.
